src/features/customerView/CustomerViewContainer.tsx

```diff
@@ -41,6 +41,7 @@ const CustomerViewContainer = () => {
   const leases = getLeases(data.profile);
   const boats = getBoats(data.profile);
   const applications = getApplications(data.profile, data.boatTypes || []);
+  const openBills: BerthBill[] = getOpenBills(data.profile);
```

Comment:
Doesn't TS infer `BerthBill[]` type here?

Reply:
Yes, removed. Was added there just to test an empty list and forgot to remove it 😞